### PR TITLE
#6Task. Nazar Lukyanovich

### PR DESCRIPTION
**Pull request by nazlukyanovich**
https://travis-ci.org/github/nazlukyanovich/js-assignments/builds/717854947

### CODE REVIEW

File: task/06-conditions-n-loops-tasks.js [hunk 1]
```diff
@@ -30,7 +30,12 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 3 == 0) {
+        if (num % 5 == 0) return 'FizzBuzz'
+        return 'Fizz'
+    }
+    if (num % 5 == 0) return 'Buzz';
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: "operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"

File: task/06-conditions-n-loops-tasks.js [hunk 2]
```diff
@@ -431,7 +509,19 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let check = function(x1, x2, x3) {
+        return x1 === x2 && x2 === x3 && x1 != undefined;
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: "operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
"0" != 0
and "0" !== 0)"

**Review by vramaniuk — COMMENTED** [review]
[Please, try to follow these rules](https://javascript.info/coding-style)

**Comment by vramaniuk** [comment]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by nazlukyanovich** [comment]
On my last commit, all operators were already strict.

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**